functions.gs

```diff
@@ -285,3 +285,22 @@ function OCTOPART_GET_DESCRIPTION(mpn_or_sku, manuf) {
      return "(no description retrieved)";
   }
 }
+
+/**
+ * @param {string} mpn_or_sku is the search term. Search for components by manufacturer and/or part number. Part number terms may contain wildcards (“*”) but must also contain at least three non-wildcard characters.
+ * @param {string} manuf limits the search result to the specified manufacturer, if desired (default: no limitation)
+ * @returns {string}
+ * @customfunction
+ */
+function OCTOPART_GET_IMAGE(mpn_or_sku, manuf) {
```

Comment:
Hello,

I put to If in order to prioritize large image. I don't really know what you mean by "wrong indent" ?? 
I used to code for my specifications, it's true that a customizable function would be better, but that's my first use of JS ever, so i don't think that it will be very optimized... I'll try.
That's the changed i've done, do they look good ? 

this.getImageSets = function() {
 

    for(var i=0; i<this._part.imagesets.length;i++){
      
        if(this._part.imagesets[i].large_image != null)
          return this._part.imagesets[i].large_image.url;       
       
    }
    for(var j=0; j<this._part.imagesets.length;j++){
      
        if(this._part.imagesets[j].medium_image != null)
          return this._part.imagesets[j].medium_image.url;
       
        
    }
    throw "No images found."
  }